mod-source-record-manager-server/src/main/resources/rules/marc_bib_rules.json

```diff
@@ -69,6 +69,60 @@
           ]
         }
       ]
+    },
+    {
+      "target": "dates.dateTypeId",
+      "description": "Date type ID",
+      "subfield": [],
+      "createSingleObject": true,
+      "rules": [
+        {
+          "description": "",
```

Comment:
can we not specify this empty field?

Reply:
@RuslanLavrov  Oh I will just remove this field, Thank you!

Reply:
got it, thank you!